docs.bruno/bruno.json

```diff
@@ -0,0 +1,5 @@
+{
+  "version": "1",
+  "name": "docs.bruno",
+  "type": "collection"
+}
\ No newline at end of file
```

Comment:
It's a good practice to add a newline at the end of the file. This can prevent potential issues with some tools that may not handle files without a newline at the end properly. [medium]